ios/App/Shared/models/local/LocalMediaProgress.swift

```diff
@@ -63,8 +63,12 @@ class LocalMediaProgress: Object, Codable {
         try container.encode(localLibraryItemId, forKey: .localLibraryItemId)
         try container.encode(localEpisodeId, forKey: .localEpisodeId)
         try container.encode(duration, forKey: .duration)
-        try container.encode(progress, forKey: .progress)
-        try container.encode(currentTime, forKey: .currentTime)
+        if progress.isNaN == false {
+            try container.encode(progress, forKey: .progress)
+        }
+        if currentTime.isNaN == false {
+            try container.encode(currentTime, forKey: .currentTime)
+        }
```

Comment:
This is honestly to fix an introduction of bad data into my database during development. I left it in, as it is desirable not to bomb out on encoding if these values are NaN.